src/chat.cpp

```diff
@@ -220,11 +215,6 @@ void ChatBuffer::scrollBottom()
 	m_scroll = getBottomScrollPos();
 }
 
-void ChatBuffer::scrollTop()
-{
-	m_scroll = getTopScrollPos();
```

Comment:
Although very short and trivial I'd prefer keeping this for consistency with `scrollBottom`

Reply:
Change reverted.